File: solutions/go/pangram/1/pangram.go
package pangram

import (
    "unicode"
    "strings"
)

func IsPangram(input string) bool {
    counts := map[rune]int{}
    for _, c := range strings.ToLower(input) {
        if unicode.IsLetter(c) {
            counts[c]++
        }
    }
	return len(counts) == 26
}

File: solutions/csharp/pangram/1/Pangram.cs
using System;
using System.Collections.Generic;
using System.Linq;

public static class Pangram
{
    static HashSet<char> Alphabet = new HashSet<char>(
        Enumerable.Range('a', 'z'-'a' + 1).Select(c => (char)c)
    );

    public static bool IsPangram(string input)
    {
        var charsInInput = new HashSet<char>(
                input
                    .ToLower()
                    .Where(Alphabet.Contains)
                );

        return charsInInput.SetEquals(Alphabet);
    }
}
